instant/flags.vim

```diff
@@ -51,3 +51,11 @@ call s:plugin.Flag('clang_format_style', 'file')
 " "goimports" (https://godoc.org/golang.org/x/tools/cmd/goimports) to
 " additionally adjust imports when formatting.
 call s:plugin.Flag('gofmt_executable', 'gofmt')
+
+""
+" The path to the js-beautify executable.
+call s:plugin.Flag('js_beautify_executable', 'js-beautify')
+
+""
+" Default formatter for javascript files, 'clang_format' or 'js_beautify'.
+call s:plugin.Flag('javascript_formatter', 'clang_format')
```

Comment:
I'm inclined to say we don't need this flag. The default is just the default. If someone has a specific preference they can configure it using `b:codefmt_formatter` and autocmds. I would like to add some recipes for that into the codefmt docs, though.

There is one point worth mentioning: Registration order matters for formatters. As-is, clang-format will take precedence since it's registered before js-beautifier, but my guess is it shouldn't since it doesn't specialize in JS while js-beautify does.

I will probably want a blacklist flag to say "I don't have a specific formatter I want, but formatter X is annoying and I know I never want it by default".